dcpy/builds/load.py

```diff
@@ -63,6 +63,8 @@ def import_dataset(
     else:
         preproc_func = None
 
+    if pg_client and (not ds.destination):
+        ds.destination = plan.InputDatasetDestination.postgres
```

Comment:
Really this is to make the cli command to import a single dataset simpler. Doesn't make a ton of sense to import a dataset to anything other than pg database